app/Imports/ImportCategory.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace App\Imports;
+
+use Illuminate\Support\Collection;
+use Maatwebsite\Excel\Concerns\ToModel;
+use App\Models\Category;
+use App\Models\Product;
+use App\Models\CategoryProduct;
+use App\Models\AttributeCategory;
+use App\Models\Attribute;
+use App\Models\ProductAttribute;
+
+class ImportCategory implements ToModel
+{
+   
+   public function model(array $row)
+   {  
+	   if($row[2]!='Item_Name' && $row[2]!='' && isset($row[5])){
+		    if($row[1]=='MODEL'){ $product_type="Single"; }else{ $product_type="Option"; }
+			//check product exist or not
+		   $product=Product::where('name',$row[2])->where('product_type',$product_type)->first();
+		   if(isset($product['id'])){
+				   $product_id=$product['id'];
+			   }else{
+				  
+				   //Add product
+				   $product= Product::create([
+					   'name' => $row[2],
+						'product_type' => $product_type,
+					   'is_deleted' => 0,
+					   'overview' => $row[54]??'',
+					   'sizing' => $row[55]??'',
+					   'instruction_of_use' => $row[53]??'',
+				   ]);
+				    
+				   $product_id=$product['id'];
+			   }
+		   $cat_arr=explode(',',$row[5]);
+		   
+		   if(count($cat_arr)>0){
+		   //Add multiple category
+		   foreach($cat_arr as $k=>$v){
```

Comment:
@kams2391 Please check if we can use Heading row here: https://docs.laravel-excel.com/3.1/imports/heading-row.html

Reply:
It is done.